docs/SNACKBAR.md

```diff
@@ -0,0 +1,97 @@
+# NearIt-UI for missing permissions alert
+
+Your app should ask the user to grant location permission and to turn on the bluetooth (if your use case implies beacons).
+
+If you want to show a persistent snackbar that alert the user if any permission is missing, this library provides a convenient `View`.
+This snackbar visually specifies what is missing (by showing/hiding the two icons) and will automatically hide itself when every permission has been granted.
+
+![missing_both](missing_both.png)
+![missing_bt](missing_bt.png)
+![missing_loc](missing_loc.png)
+
+Add the following xml element where you want to show the snackbar
+
+```xml
+<com.nearit.ui_bindings.permissions.views.PermissionSnackbar
+        android:id="@+id/permission_snackbar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content" />
+```
+
+and keep a reference in your activity
+
+```java
+PermissionSnackbar snackbar = (PermissionSnackbar) findViewById(R.id.permission_snackbar);
+```
+
+It is **important** that you pass an Activity reference to the snackbar: by doing this, clicking the snackbar button will cause the launch of the permissions request flow.
+
+![snackbar](snackbar.gif)
+
+To set the activity use the following method:
+
+```java
+snackbar.setActivity(YourActivity.this, NEAR_PERMISSION_REQUEST);
+```
+
+where `NEAR_PERMISSION_REQUEST` is an int value defined by you that will identify the request.
+Then in your activity you should catch the `result` by referring to the same request code.
+
```

Comment:
Add a sample of the callback method: should I catch the onActivityResult or the onRequestPermissionsResult? It's not clear from the doc.

Reply:
done and updated the sample